swan-lake/integration-tools/openapi-tool.md

```diff
@@ -117,11 +170,22 @@ $ bal openapi -i hello.yaml --tags "pets", "list"
 
 Once you execute the command, only the operations related to the given tags get included in the generated service file.
 
+## Generate Ballerina service contract type from OpenAPI Contracts
+
+A service contract-first approach is introduced as a standout key feature using the OpenAPI tool and the HTTP package. By generating a service contract type directly from the OpenAPI contract, you can design services that are contract-compliant from the outset. 
```

Comment:
IMO, things like "A service contract-first approach is introduced as a standout key feature using the OpenAPI tool and the HTTP package." are for the release note, not for generic documentation.